### ISSUE TITLE
Some fields may not be read correctly

### ISSUE THREAD

**Issue by Riduidel** [feed-rs/feed-rs#2]
I'm using feed_rs to parse content of BrotherBircks web site.

Items in their RSS feed are defined this way

<item>
	<title>Cadet Thrawn outwits his opponents in the metallurgy lab</title>
	<link>http://feedproxy.google.com/~r/TheBrothersBrick/~3/eWF3-ZnktaM/</link>
	<comments>https://www.brothers-brick.com/2019/02/08/cadet-thrawn-outwits-his-opponents-in-the-metallurgy-lab/#respond</comments>
	<pubDate>Fri, 08 Feb 2019 14:00:19 +0000</pubDate>
	<dc:creator><![CDATA[Daniel]]></dc:creator>
			<category><![CDATA[LEGO]]></category>
	<category><![CDATA[Models]]></category>
	<category><![CDATA[Diorama]]></category>
	<category><![CDATA[RebelLUG]]></category>
	<category><![CDATA[Star Wars]]></category>
	<category><![CDATA[Thrawn]]></category>

	<guid isPermaLink="false">https://www.brothers-brick.com/?p=171427</guid>
	<description><![CDATA[This detailed scene by CRCT Productions depicts the famous Grand Admiral Thrawn in his early days as an Imperial cadet.]]></description>
			<content:encoded><![CDATA[CONTENT IS REDACTED as GitHub interprets it as valid HTML]]></content:encoded>
		<wfw:commentRss>https://www.brothers-brick.com/2019/02/08/cadet-thrawn-outwits-his-opponents-in-the-metallurgy-lab/feed/</wfw:commentRss>
	<slash:comments>0</slash:comments>
<post-id xmlns="com-wordpress:feed-additions:1">171427</post-id>	<feedburner:origLink>https://www.brothers-brick.com/2019/02/08/cadet-thrawn-outwits-his-opponents-in-the-metallurgy-lab/</feedburner:origLink>
</item>

And the content can't be read. The full RSS feed looks like https://www.brothers-brick.com/feed/

**Comment by markpritchard** [comment]
Hi,

I made lots of changes in the fix for #7 and introduced a little command line tool. I tried it with https://www.brothers-brick.com/feed/ and it works fine now:

```bash
$ echo "https://www.brothers-brick.com/feed/" | cargo run --bin testurls
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/testurls`
https://www.brothers-brick.com/feed/  ... ok
```

Let me know if you still have problems.

**Comment by markpritchard** [comment]
I'll close this issue for now, but feel free to re-open if you still have problems.
